http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/db/RuntimeConfigQueries.scala

```diff
@@ -0,0 +1,63 @@
+package org.broadinstitute.dsde.workbench.leonardo
+package db
+
+import java.time.Instant
+
+import org.broadinstitute.dsde.workbench.leonardo.db.LeoProfile.api._
+import org.broadinstitute.dsde.workbench.leonardo.db.LeoProfile.mappedColumnImplicits._
+
+import scala.concurrent.ExecutionContext
+
+object RuntimeConfigQueries {
+  type ClusterJoinLabelJoinRuntimeConfig = Query[((ClusterTable, Rep[Option[LabelTable]]), Rep[Option[RuntimeConfigTable]]), ((ClusterRecord, Option[LabelRecord]), Option[RuntimeConfigRecord]), Seq]
+
+  val runtimeConfigs = TableQuery[RuntimeConfigTable]
+
+  val clusterLabelRuntimeConfigQuery: ClusterJoinLabelJoinRuntimeConfig = {
+    for {
+      (cluster, label) <- clusterQuery
```

Comment:
I feel like the types are a bit off here -- shouldn't it be `((cluster, label), runTimeConfig) <- …`? Also this may have the same problem with joining-before-filtering… I'm playing around with it, might PR to this branch with a commit.

Reply:
https://github.com/DataBiosphere/leonardo/pull/1276/